tools/clang/lib/SPIRV/SPIRVEmitter.cpp

```diff
@@ -4363,7 +4391,8 @@ uint32_t SPIRVEmitter::processIntrinsicModf(const CallExpr *callExpr) {
     }
   }
 
-  emitError("Unknown argument type passed to Modf function.");
+  emitError("unknown argument type passed to Modf function",
```

Comment:
invalid argument type passed to Modf intrinsic function?

Reply:
Done